src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
```

Comment:
good now you need to use variables!
all math actions or dials which you use 2 or more times should be used as variables!